android/src/main/kotlin/com/hypertrack/sdk/flutter/HypertrackError.kt

```diff
@@ -0,0 +1,15 @@
+package com.hypertrack.sdk.flutter
+
+// enum naming convention is ingored to make datatype sync
+// across platforms easier
+enum class HypertrackError {
```

Comment:
```suggestion
enum class HyperTrackError {
```

Reply:
It's better to have HyperTrack.Error than HyperTrackError

Reply:
In that case it's necessary to have it in one file. I think it's better to separate it to have better structure

Reply:
And this class is not the part of public API